### ISSUE TITLE
Incorrect call to authClient.accessToken()

### ISSUE THREAD

**Issue by gabmontes** [bloq/cloud-sdk#7]
The call to `authClient.accessToken()` expects to params and returns a promise:

https://github.com/bloq/cloud-sdk/blob/3f4e172830460f0b055b333329cb99d89a23660c/src/auth/index.js#L21

Actual code sends a callback:

https://github.com/bloq/cloud-sdk/blob/3f4e172830460f0b055b333329cb99d89a23660c/src/insight/socket.js#L30

**Comment by ndelvalle** [comment]
Yes @gabmontes you are right. Socket.io is not yet fully implemented in this SDK. I want to have the same regular socket.io client API but using the cloud auth under the hood and make it transparent to the user.

That is why I did not add any of this in the readme.

**Comment by gabmontes** [comment]
If it is not fully implemented, a cleaner option is to have it in a branch and avoid even mentioning in the documentation or exposing it in the API.
